src/App.jsx

```diff
@@ -1,11 +1,40 @@
-import Navbar from './components/Navabar/Navbar'
+// eslint-disable-next-line no-unused-vars
+import React from "react";
+import { 
+  Route, 
+  RouterProvider, 
+  createBrowserRouter,
+  createRoutesFromElements,
+ } from "react-router-dom";
+import Home from "./pages/Home";
+import Football from "./pages/Football"
+import Fone from "./pages/F1"
+import Golf from "./pages/Golf"
+import Nba from "./pages/Nba"
+import Ufc from "./pages/Ufc"
+import Swimming from "./pages/Swimming";
+import RootLayout from "./layout/RootLayout";
+
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route path="/" element={<RootLayout/>}>
```

Comment:
Don't use this name RootLayout. It's confusing. you can use Layout

Reply:
Done